skimage/feature/_hog.py

```diff
@@ -61,7 +61,10 @@ def hog(image, orientations=9, pixels_per_cell=(8, 8), cells_per_block=(3, 3),
            For details, see [3]_, [4]_.
 
     visualise : bool, optional
-        Also return an image of the HOG.
+        Also return an image of the HOG.  For each cell and orientation bin,
+        the image contains a line segment that is centered at the cell center,
+        is perpendicular to the bisector of the orientation bin, and has
```

Comment:
It's simply the midpoint.  As you can see in [this](https://github.com/scikit-image/scikit-image/issues/2453#issuecomment-272940067) example, the line segments are indeed perpendicular to the midpoints of orientation bins (they all have slope 1 or -1).

I've updated the docstring; let me know if there's anything else you'd like me to change.